wcomponents-core/src/main/java/com/github/bordertech/wcomponents/DefaultWComponent.java

```diff
@@ -20,8 +18,6 @@ public void setTemplate(final String aUrl) {
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.